app/components/DocsPopover.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+import { offset, useFloating } from '@floating-ui/react'
+import { Popover } from '@headlessui/react'
```

Comment:
Are we testing the `@headlessui` alternative to the one we're using elsewhere? It's probably a good idea because we were seeing some performance issues with radix.

Reply:
Yeah, I have this hunch that headless is just better overall. I didn’t realize we were using Radix Popover already — looks like it’s only for the date picker. Would probably be pretty easy to switch it over.

Reply:
Looked into it and it's not Radix, it's React Aria's `usePopover`, which is pretty well integrated with their date picker stuff (the only context we use it). So I don't think it's worth teasing apart.

https://github.com/oxidecomputer/console/blob/e8e4f576e6ee5c3e3544eaf5230a71e07c79a428/app/ui/lib/Popover.tsx